CLAUDE.md

```diff
@@ -0,0 +1,141 @@
+# CLAUDE.md

```

Comment:
Great to have this documented indeed :+1: 

Taking the source code repositories I know that don't have a README file, but that do have other textual files in the root, asquare is definitely winning the gold medal for me :smile: ...